examples/dt/dcn/service-values.yaml

```diff
@@ -0,0 +1,261 @@
+# local-config: referenced, but not emitted by kustomize
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  preserveJobs: false
+  cinderAPI:
+    replicas: 3
+    customServiceConfig: |
+      [DEFAULT]
+        default_availability_zone = ceph
+  cinderBackup:
+    replicas: 3
+    customServiceConfig: |
+      [DEFAULT]
+      backup_driver = cinder.backup.drivers.ceph.CephBackupDriver
+      backup_ceph_pool = backups
+      backup_ceph_user = openstack
+  cinderVolumes:
+    ceph:
+      customServiceConfig: |
+        [DEFAULT]
+        enabled_backends = ceph
+        [ceph]
+        volume_backend_name = ceph
+        volume_driver = cinder.volume.drivers.rbd.RBDDriver
+        rbd_ceph_conf = /etc/ceph/ceph.conf
+        rbd_user = openstack
+        rbd_pool = volumes
+        rbd_flatten_volume_from_snapshot = False
+        rbd_secret_uuid = CHANGEME
+        rbd_cluster_name = ceph
+        backend_availability_zone = ceph
```

Comment:
Based on the offline discussion, let's change this to az0 which would be the name unified with compute Availability zone of the default site's compute nodes where this ceph cluster is colocated.
The name of volume and compute AZ must be the same otherwise a similar ERROR will appear:
2024-10-07 09:31:53.192 2 ERROR nova.compute.manager [None req-b437ddf9-b83b-421e-ae44-2756d6392706 958be7b3096d42f28e2c35e6752e5dcb e308ae3b29eb4b9194cf5bbd10536d30 - - default default] [instance: 00528f0a-4fc2-4b06-8aa5-357c504da23c] Failed to attach 6e6af3a3-be24-4c67-9ba8-7b069b1bde7c at /dev/vdb: nova.exception.InvalidVolume: Invalid volume: Instance 00528f0a-4fc2-4b06-8aa5-357c504da23c and volume 6e6af3a3-be24-4c67-9ba8-7b069b1bde7c are not in the same availability_zone. Instance is in az0. Volume is in ceph

Reply:
This has been addressed.